components/accordion/index.css

```diff
@@ -11,17 +11,295 @@ governing permissions and limitations under the License.
+		var(--spectrum-gray-900-rgb),
+		var(--spectrum-background-opacity-default)
+	);
+	--spectrum-accordion-background-color-hover: rgba(
+		var(--spectrum-gray-900-rgb),
+		var(--spectrum-background-opacity-hover)
+	);
+	--spectrum-accordion-background-color-down: rgba(
+		var(--spectrum-gray-900-rgb),
+		var(--spectrum-background-opacity-down)
 	);
-	--spectrum-accordion-item-title-padding-y: var(
-		--spectrum-global-dimension-size-150
+	--spectrum-accordion-background-color-key-focus: rgba(
+		var(--spectrum-gray-900-rgb),
+		var(--spectrum-background-opacity-key-focus)
 	);
-	--spectrum-accordion-animation-duration: var(
-		--spectrum-global-animation-duration-100
+	/* Label */
+	--spectrum-accordion-item-header-color-default: var(
+		--spectrum-neutral-content-color-default
+	);
+	--spectrum-accordion-item-header-color-hover: var(
+		--spectrum-neutral-content-color-hover
+	);
+	--spectrum-accordion-item-header-color-down: var(
+		--spectrum-neutral-content-color-down
+	);
+	--spectrum-accordion-item-header-color-key-focus: var(
+		--spectrum-neutral-content-color-key-focus
+	);
+
+	--spectrum-accordion-item-header-disabled-color: var(
+		--spectrum-disabled-content-color
+	);
+	--spectrum-accordion-item-content-disabled-color: var(
+		--spectrum-disabled-content-color
+	);
+	/* Body */
+	--spectrum-accordion-item-content-color: var(--spectrum-body-color);
+	/* Focus indicator */
+	--spectrum-accordion-focus-indicator-color: var(
+		--spectrum-focus-indicator-color
+	);
+	/* Divider */
+	--spectrum-accordion-divider-color: var(--spectrum-gray-300);
+	--spectrum-accordion-min-block-size: max(
+		var(--mod-accordion-item-height, var(--spectrum-accordion-item-height)),
+		calc(
+			var(
+					--mod-accordion-item-header-top-to-text-space,
+					var(--spectrum-accordion-item-header-top-to-text-space)
+				) +
+				var(
+					--mod-accordion-item-header-bottom-to-text-space,
+					var(--spectrum-accordion-item-header-bottom-to-text-space)
+				) +
+				(
+					var(
+							--mod-accordion-item-header-font-size,
+							var(--spectrum-accordion-item-header-font-size)
+						) *
+						var(
+							--mod-accordion-item-header-line-height,
+							var(--spectrum-accordion-item-header-line-height)
+						)
+				)
+		)
+	);
+}
+
+.spectrum-Accordion--compact {
+	&.spectrum-Accordion--sizeS {
+		--spectrum-accordion-item-height: var(--spectrum-component-height-75);
+		--spectrum-accordion-item-header-top-to-text-space: var(
+			--spectrum-accordion-top-to-text-compact-small
+		);
+		--spectrum-accordion-item-header-bottom-to-text-space: var(
+			--spectrum-accordion-bottom-to-text-compact-small
+		);
+	}
+
+	&.spectrum-Accordion--sizeM {
+		--spectrum-accordion-item-height: var(--spectrum-component-height-100);
+		--spectrum-accordion-item-header-top-to-text-space: var(
+			--spectrum-accordion-top-to-text-compact-medium
+		);
+		--spectrum-accordion-item-header-bottom-to-text-space: var(
+			--spectrum-accordion-bottom-to-text-compact-medium
+		);
```

Comment:
I would move these under `.spectrum-Accordion--compact`, to be the default for this density.